### ISSUE TITLE
Terms of Service - "I agree" button needs to be more specific

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3977]
## Area or Component
Patient Experience

## Quick Summary
The button for terms of service is labelled "I agree" and the helper text above is "By agreeing, you consent to our terms of service." On its own, the aria label for the button is not descriptive enough.

## Accessibility Violation
The element's accessible name is missing or is not appropriate for the element's purpose. https://axe.deque.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
Update the label to "I agree with the terms of service".

Question for the design/content team: Should the entire button be updated? Or just the accessible label?

## Screenshots
<img width="480" alt="Screen Shot 2022-07-14 at 4 59 35 PM" src="https://user-images.githubusercontent.com/80282552/179120483-398f6d14-b9b9-4dba-b133-f35813197ac1.png">

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/timeOfTest/TermsOfService.tsx#L58-L58

## Original Deque Issue
aria-name-missing-incorrect | The element's accessible name is missing or is not appropriate for the element's purpose. | "I agree" button is not an accessible name; label should likely be "i agree with the terms of service" instead | Name: The element's name is missing or incorrect | https://axe.deque.com/rule-help/en/aria-name-missing-incorrect | critical | Yes | No | Interactive Elements | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-base-lightest:nth-of-type(3) > main.patient-app.padding-bottom-4.bg-base-lightest.padding-top-05:nth-of-type(1) > form.grid-container.maxw-tablet.usa-prose:nth-of-type(1) > div.tos-content.prime-formgroup.usa-prose.height-card-lg.overflow-x-hidden.font-body-3xs:nth-of-type(1) > p:nth-of-type(10) > a:nth-of-type(1)"] | The element's accessible name is missing or is not appropriate for the element's purpose. | <a href="https://www.cms.gov/Research-Statistics-Data-and-Systems/Computer-Data-and-Systems/Privacy/PrivacyActof1974.html" data-axe-pro-tab-stop-index="6">click here</a> | wcag2a,wcag412b | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:04:27.195Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by emmastephenson** [comment]
We have options here - we could either change the button for all users, or give it a more descriptive `aria-labelledby` that links to the text above ("By agreeing, you consent to our terms of service")

**Comment by emmastephenson** [comment]
Design decision in 7/22 meeting was for the last option - give the button an aria-labelled by that links to the text above it.
